lib/utils/fs.go

```diff
@@ -280,6 +345,9 @@ func overwriteFile(filePath string) (err error) {
 	}
 
 	_, err = io.CopyN(f, rand.Reader, size)
+
+	_ = f.Sync() // sync to ensure commit to hardware
```

Comment:
Should we be ignoring an error here?

Writes are buffered in many cases, meaning the error that we are checking (from `CopyN`) can be nil and the real error would surface here, but we're ignoring it.